src/components/KDropdownMenu/KDropdownMenu.vue

```diff
@@ -170,6 +174,16 @@ const handleSelection = (item: DropdownItem): void => {
   selectedItem.value = item
 }
 
+const handleCloseDropdown = (): void => {
+  hidePopover.value = true
+
+  // reset the hidePopover value so it's ready for the next time the dropdown is opened
+  // need nextTick to ensure the popover is hidden before resetting the value
+  nextTick(() => {
+    hidePopover.value = undefined
+  })
```

Comment:
issue: This `nextTick` isn't doing anything unless you `await`

What's the purpose here?

Reply:
Good call. The purpose is to reset it to initial value. Even though it clearly worked without `await`, still kept the `nextTick` and added the `await` to avid possibility of race condition.